packages/web/pages/raids.tsx

```diff
@@ -188,14 +190,21 @@ const RaidsPage: React.FC = () => {
           background="whiteAlpha.50"
           borderRadius={10}
           p={7}
-          gap={6}
-          flexDir="column"
+          gap={{ base: 10, md: 6 }}
+          flexDir={{ base: 'row', md: 'column' }}
+          pos="relative"
         >
-          <Flex gap={6}>
+          <Flex
+            gap={6}
+            flexDir={{
+              base: 'column',
+              md: 'row',
+            }}
+          >
             {phases.map(({ title, subtitle, description }, index) => (
               <Box key={title}>
                 <Flex alignItems="baseline" justifyContent="center">
-                  <Text fontWeight={700} fontSize={36}>
+                  <Text fontWeight={700} fontSize={36} textAlign="center">
```

Comment:
i reduced the title size on mobile 👍
do you think we should remove or reduce the size of the extra guttering on all cards throughout the app on mobile? it would be good to keep it consistent if so. can we get the opinion of @peth-yursick @davort ?

Reply:
looks better with the reduced padding: 
<img width="504" alt="Screenshot 2022-11-26 at 10 00 14" src="https://user-images.githubusercontent.com/35112939/204080929-a6b56b5d-8ca3-4d7a-a467-ae1f3f348a45.png">

Reply:
> i reduced the title size on mobile +1 do you think we should remove or reduce the size of the extra guttering on all cards throughout the app on mobile? it would be good to keep it consistent if so. can we get the opinion of @peth-yursick @davort ?

I'd say yes, reduce. I was originally thinking it should be removed on mobile but reducing the gutters looks to work pretty well.